### DISCUSSION TITLE
启动器图标调到最小，图标之间间隔太大

### DISCUSSION THREAD

**Discussion by a137933** [linuxdeepin/developer-center#2953] [Features Request & Ideas | 特性请求 & 头脑风暴]
如题，左右或者上下两个图标之间的间隔甚至可以再放一个图标。
能改成这样吗？
比如，图标最大时：一页的图标数量4行7列。
           图标最小时：一页的图标数量变成8行14列
![image](https://user-images.githubusercontent.com/34773099/109091613-19456780-7750-11eb-8caf-70655e4b789e.png)

**Comment by a137933** [comment]
？？？

**Comment by meiyixiang** [comment]
Thank you for your suggestion, it should be improved in terms of interaction and interface display aesthetics
